pkgs/top-level/all-packages.nix

```diff
@@ -19303,6 +19303,8 @@ with pkgs;
 
   whatsapp-for-linux = callPackage ../applications/networking/instant-messengers/whatsapp-for-linux { };
 
+  whatsie = libsForQt5.callPackage ../applications/networking/instant-messengers/whatsie { };
```

Comment:
It's `libsForQt5.full`

Reply:
Usually we specify the packages we are using, instead of relying in bundles.
The bundles are useful for development, not so much for inclusion in Nixpkgs.

Reply:
From what I noticed many packages in `all-packages.nix` use this, that's why I included it. I'm not that fluent in nix yet, but using standard `callPackage` wouldn't mean to prefix all qt references with `pkgs` etc.?